wsmatrix@martin.zurowietz.de/ThumbnailsBoxOverride.js

```diff
@@ -189,14 +207,13 @@ var ThumbnailsBoxOverride = class {
 
       let thumbnailHeight = portholeHeight * this._scale;
       let thumbnailWidth = Math.round(portholeWidth * this._scale);
-      let thumbnailBoxWidth = thumbnailWidth * this.getColumns() + spacing * (this.getColumns() - 1);
       let roundedHScale = thumbnailWidth / portholeWidth;
 
       let slideOffset; // X offset when thumbnail is fully slid offscreen
       if (rtl)
-         slideOffset = - (thumbnailBoxWidth + themeNode.get_padding(St.Side.LEFT));
+         slideOffset = - (thumbnailWidth + themeNode.get_padding(St.Side.LEFT));
```

Comment:
`thumbnailBoxWidth` should still be correct here because we have a 2D grid instead of a single column of thumbnails. It doesn't have any effect, though, since the thumbnails box in the overview does not seem to slide out of the display any more.